TESTS WITH ERRORS 
test06_GM_8Puzzle_02 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 140, in test06_GM_8Puzzle_02
    p8.read('puzzle8_center_empty.txt')
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\game_master.py", line 134, in read
    for fr in read_tokenize(final_path):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\read.py", line 37, in read_tokenize
    for line in file:
  File "C:\Program Files\Python37\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 257: character maps to <undefined>

test08_GM_8Puzzle_04 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 185, in test08_GM_8Puzzle_04
    p8.read('puzzle8_center_empty.txt')
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\game_master.py", line 134, in read
    for fr in read_tokenize(final_path):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\read.py", line 37, in read_tokenize
    for line in file:
  File "C:\Program Files\Python37\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 257: character maps to <undefined>

test16_DFS_8Puzzle_02 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 338, in test16_DFS_8Puzzle_02
    p8.read('puzzle8_center_empty.txt')
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\game_master.py", line 134, in read
    for fr in read_tokenize(final_path):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\read.py", line 37, in read_tokenize
    for line in file:
  File "C:\Program Files\Python37\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 257: character maps to <undefined>

test17_DFS_8Puzzle_03 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 357, in test17_DFS_8Puzzle_03
    p8.read('puzzle8_center_empty.txt')
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\game_master.py", line 134, in read
    for fr in read_tokenize(final_path):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\read.py", line 37, in read_tokenize
    for line in file:
  File "C:\Program Files\Python37\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 257: character maps to <undefined>

test20_BFS_8Puzzle_03 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 414, in test20_BFS_8Puzzle_03
    p8.read('puzzle8_center_empty.txt')
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\game_master.py", line 134, in read
    for fr in read_tokenize(final_path):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\read.py", line 37, in read_tokenize
    for line in file:
  File "C:\Program Files\Python37\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 257: character maps to <undefined>

-------------------------------
TESTS FAILED  
test12_BFS_Hanoi_01 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 278, in test12_BFS_Hanoi_01
    [22, ((2,), (3,), (1,))],
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 55, in runPlayXSteps
    self.assertEqual(results[index], expected)
AssertionError: Tuples differ: ((1, 3), (2,), ()) != ((), (3,), (1, 2))

First differing element 0:
(1, 3)
()

- ((1, 3), (2,), ())
+ ((), (3,), (1, 2))

test13_BFS_Hanoi_02 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 296, in test13_BFS_Hanoi_02
    [23, ((5,), (4,), (1, 2, 3))],
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 55, in runPlayXSteps
    self.assertEqual(results[index], expected)
AssertionError: Tuples differ: ((2, 4, 5), (1,), (3,)) != ((4, 5), (1,), (2, 3))

First differing element 0:
(2, 4, 5)
(4, 5)

- ((2, 4, 5), (1,), (3,))
?   ---               -

+ ((4, 5), (1,), (2, 3))
?                 +++


-------------------------------
TESTS SKIPPED 
-------------------------------
